rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_cooling_type_fluid_loop.py

```diff
@@ -0,0 +1,42 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.utils.jsonpath_utils import find_exactly_one_with_field_value
+
+COOLING_SYSTEM_TYPE = schema_enums["CoolingSystemOptions"]
+
+
+def is_hvac_sys_cooling_type_fluid_loop(rmi_b, hvac_b_id):
+    """Returns TRUE if the HVAC system has fluid_loop cooling. Returns FALSE if the HVAC system has anything other
+    than fluid_loop cooling or if it has more than 1 cooling system.
+
+    Parameters
+    ----------
+    rmi_b : json
+        RMD at RuleSetModelInstance level
+    hvac_b_id : str
+        The HVAC system ID.
+
+    Returns
+    -------
+    bool
+        True: HVAC system has fluid_loop cooling
+        False: HVAC system has a cooling system type other than fluid_loop
+    """
+    is_hvac_sys_cooling_type_fluid_loop_flag = False
+
+    # Get the hvac system
+    hvac_b = find_exactly_one_with_field_value(
+        "$.buildings[*].building_segments[*].heating_ventilation_air_conditioning_systems",
+        "id",
+        hvac_b_id,
+        rmi_b,
+    )
+    # Check if hvac_b has preheat system
+    cooling_system = hvac_b.get("cooling_system")
+    if cooling_system:
```

Comment:
This is correct, but could be more succint:
```
is_hvac_sys_cooling_type_fluid_loop_flag =
    cooling_system and 
    cooling_system.get("chilled_water_loop") and
    cooling_system["cooling_system_type"] == COOLING_SYSTEM_TYPE.FLUID_LOOP
```
And then line 24 could also be deleted.

Reply:
It seems:
```
cooling_system and 
    cooling_system.get("chilled_water_loop") and
    cooling_system["cooling_system_type"] == COOLING_SYSTEM_TYPE.FLUID_LOOP
```
returns `None` if `cooling_system` is `None` or `cooling_system.get("chilled_water_loop")` is `None`, which cannot be assign to the flag.

I was thinking to flip the default flag to `True` so the logic can be written in a more clean way, but give up because I don't want to alter the RDS.

Reply:
Good point about returning None.
How about:
``` 
cooling_system is not None and
cooling_system.get("chilled_water_loop") is not None and
cooling_system["cooling_system_type"] == COOLING_SYSTEM_TYPE.FLUID_LOOP
```
?

Reply:
True, this will work. I can condense the if statement to the one you proposed. Seems cleaner to me.